design/authenticators-status/authn-status-new-implem.md

```diff
@@ -0,0 +1,86 @@
+# Implementing Authenticator Status Checks
+The authenticator status check allows operators to check if the authenticator is configured correctly. With this capability, 
+operators can make a RESTful request using the Conjur API and receive a response identifying if configuration was successful.
+If not, the operators receive the proper error identifying the step of failure. 
+The first authenticator to utilize this feature is Authn OpenID Connect. 
+
+_For a complete status check flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+
+This doc describes how to expand the status check functionality to other authenticators.
+For each authenticator, a suite of [general](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn_status_general.md) 
+checks are run followed by those specific to the authenticator.  
+
+General checks include:
+1. The requesting user has access to the authenticator status route webservice
```

Comment:
please add a link to the security section of the epic where we explain what this webservice is. also, i don't like the word `route` here as we grant permissions to a webservice as a conjur resource